### ISSUE TITLE
False positive in LiteralInInterpolation cop

### ISSUE THREAD

**Issue by sferik** [bbatsov/rubocop#2959]
The string:

```ruby
"Between #{starts_at..ends_at}."
```

Will be autocorrected to:

```ruby
"Between starts_at..ends_at."
```

Which is not the same thing.

**Comment by alexdowad** [comment]
Just added a fix to my open PR.
